engelsystem/src/main/java/info/metadude/android/eventfahrplan/engelsystem/loading/ShiftsLoading.kt

```diff
@@ -32,7 +32,7 @@ internal object ShiftsLoading {
             ShiftsResult.Success(shifts)
         }
     } else {
-        val httpStatusMessage: String? = message()
+        @Suppress("RedundantNullableReturnType") val httpStatusMessage: String? = message()
```

Comment:
In my Android Studio, I don't get a warning here when I remove the suppress. Is this suppress for some tool or is there something I don't understand?

Reply:
Lint is showing the warning to me in Android Studio 2021.2.1 Beta 4:

![lint](https://user-images.githubusercontent.com/144518/159574519-1e7345b2-8aeb-480e-b532-313f8a57c611.png)

Reply:
It does not show up in `Android Studio Bumblebee | 2021.1.1 Patch 2`. Let's drop it.